sample/java/build.gradle

```diff
@@ -43,17 +38,15 @@ android {
     productFlavors {
         staging {
             dimension "version"
-            buildConfigField("String", "DD_OVERRIDE_LOGS_URL", "\"${project.findProperty("DD_OVERRIDE_LOGS_URL")}\"")
-            buildConfigField("String", "DD_OVERRIDE_TRACES_URL", "\"${project.findProperty("DD_OVERRIDE_TRACES_URL")}\"")
-            buildConfigField("String", "DD_OVERRIDE_RUM_URL", "\"${project.findProperty("DD_OVERRIDE_RUM_URL")}\"")
-            buildConfigField("String", "DD_RUM_APPLICATION_ID", "\"${project.findProperty("DD_RUM_APPLICATION_ID")}\"")
+            configureFlavorForSampleApp(it, rootDir)
+        }
+        production {
+            dimension "version"
+            configureFlavorForSampleApp(it, rootDir)
         }
         full {
             dimension "version"
-            buildConfigField("String", "DD_OVERRIDE_LOGS_URL", "null")
-            buildConfigField("String", "DD_OVERRIDE_TRACES_URL", "null")
-            buildConfigField("String", "DD_OVERRIDE_RUM_URL", "null")
-            buildConfigField("String", "DD_RUM_APPLICATION_ID", "null")
+            configureFlavorForSampleApp(it, rootDir)
         }
```

Comment:
What's the difference between full and production ?

Reply:
Full doesn't send anything (token and app id are empty). This was there before but IMO we should remove it and only keep staging and production. What do you think ?